src/utils.js

```diff
@@ -136,7 +136,30 @@ export const isPruneTraceOff = () =>
   );
 
 export const isSwitchedOff = () => {
-  return TracerGlobals.getTracerInputs().switchOff;
+  return TracerGlobals.getTracerInputs().switchOff || !isValidAlias();
+};
+
+export const getValidAliasesOrEmptyArray = () =>
+  process.env['VALID_ALIASES'] || [];
+
+export const getHandlerContext = () =>
+  TracerGlobals.getHandlerInputs().context || {};
+
+export const getInvokedArn = () => getHandlerContext().invokedFunctionArn || '';
+export const getInvokedVersion = () =>
+  getHandlerContext().functionVersion || '';
+
+export const getInvokedAliasOrNull = () =>
+  safeExecute(() => {
```

Comment:
Do you have a test that checks for invalid ARN?

Reply:
no, that is why it runs inside `safeExecute`

Reply:
But add a test

Reply:
added